lib/view/mapCard_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:gra_terenowa/controller/mapData_controller.dart';
+import 'package:gra_terenowa/statics/colors.dart';
+import 'package:gra_terenowa/statics/constants.dart';
+
+class MapCardPage extends StatelessWidget {
+  const MapCardPage({
+    Key? key,
+    required this.mapItemIndex,
+  }) : super(key: key);
+
+  final int mapItemIndex;
+
+  @override
+  Widget build(BuildContext context) {
+    final MapDataController _mapDataController = Get.find();
+
+    return Scaffold(
+      extendBodyBehindAppBar: true,
+      backgroundColor: AppColors.primaryWhite,
+      appBar: AppBar(
+        backgroundColor: AppColors.primaryWhite.withOpacity(Constants.opacity0),
+        foregroundColor: AppColors.primaryDark,
+        elevation: 0,
+        // title: Text(
```

Comment:
Shall we keep the commented code here and in other files and make a robust code cleaning at some point, perhaps also looking through TODOs?

Reply:
Yes, I think we should do general code cleaning, refactoring and comments before going into beta. That will of course include all the TODOs. For now, I would rather leave the commented code, since it might prove useful before cleanup. What do you think?